image/Dockerfile

```diff
@@ -16,8 +16,7 @@ FROM gcr.io/distroless/cc:nonroot as base
 WORKDIR /
 COPY ./license.html .
 COPY ./dependencies-src.zip .
-COPY --chown=nonroot:nonroot ./image/quilkin.yaml /etc/quilkin/quilkin.yaml
-COPY ./target/image/quilkin .
+COPY ./target/image/quilkin /usr/local/bin
```

Comment:
I'm fairly sure that there is no $PATH in a distroless image, since there is no shell, hence the hardcoded path in the `ENTRYPOINT`.

https://github.com/GoogleContainerTools/distroless/blob/main/README.md#entrypoints

Reply:
Yeah I know but they're still Debian images, so I wanted to see if that would work.

Reply:
😄 worth a shot!